### ISSUE TITLE
forced preview when using content search

### ISSUE THREAD

**Issue by nikodemus** [adirh3/Fluent-Search#983]
**Describe the bug**
When searching by content, the preview is always shown - even if the relevant preview module has automatic preview off. (Disabling the preview module entirely does prevent this.)

This is an issue particularly for large PDFs, where the preview is quite slow - but also if you have eg. PDFs with disturbing cover images.

This happens with both Stable and Nightly.

**To Reproduce**
1. Go to 'Preview > Modules': Native module is on, Native automatic preview is off.
2. Go to 'Apps > Files': Search using files is on.
3. Go to 'Apps > Files > File Indexing': add a directory containing PDFs to Content Indexed Search, cick on Rebuild Index.
4. Wait for indexing to finish.
5. Search for a PDF by name. Observe that the preview is not shown.
6. Search for a PDF by content. Observe that the preview is shown.

**Expected behavior**
Preview functionality should work the same way for content search and name search. (If variant behaviour is desired, it should not be a forced default.)

**Desktop (please complete the following information):**
 - Window 10/11 Version: 23H2
 - Fluent Search Version: [e.g. 1.0.0.34] 
 - Installation source: Installer

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 1.0.0.35                                          . Changelog - https://install.appcenter.ms/users/adirh3-gmail.com/apps/fluent-search-alpha/distribution_groups/exe

**Comment by nikodemus** [comment]
That fixes this for me, thank you.

I did experience some strange behaviour, where after testing different setting, preview got somehow bugged:

- Preview view was always visible after search, but either blank white or all black.
- Hitting Alt toggled the eye-indicator and sometimes made the preview appear, sometimes not.
- Toggling preview modules entirely off did not change anything.

Eventually I restarted Fluent Search, at which point it went back to normal. I'll file a proper bug report if I stumble across this again, but first attempt at reproduction didn't work.
